frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -248,9 +268,26 @@ export class Dashboard extends Component {
     }
   };
 
+  handleNavitationFromMine = () => {
+    // TODO: spent 4 hours looking for a solution to not hardcoding this scroll value. Need to find a dynamic way of scroling the screen to this location.
```

Comment:
I know this isn't your comment, but could you update this to something like `FIXME: scroll dynamically instead of by a hardcoded value`?

Reply:
looks like `handleNavitationFromMine` contains all the scroll logic, would be beneficial to rename this function to `handleScroll` or something similar, and invoke it in every situation there is scroll logic to dry up the code a bit.

Reply:
done